src/electron/device-connect-view/components/device-connect-body.tsx

```diff
@@ -1,19 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
 import { BrowserWindow } from 'electron';
 import * as React from 'react';
+
 import { DeviceConnectConnectedDevice } from './device-connect-connected-device';
 import { DeviceConnectFooter } from './device-connect-footer';
 import { DeviceConnectHeader } from './device-connect-header';
 import { DeviceConnectPortEntry, DeviceConnectPortEntryDeps } from './device-connect-port-entry';
+import { DeviceConnectState } from './device-connect-state';
 
 export type UpdateStateCallback = (newState: DeviceConnectState, deviceName?: string) => void;
 
-export enum DeviceConnectState {
-    Default,
-    Connecting,
-    Connected,
-    Error,
+export interface DeviceConnectBodyState {
```

Comment:
should this be DeviceConnectInfo instead of body state? it is not storing the state of the body, but the state of device connection.

Reply:
The idea was to follow a similar pattern as the one we have with deps. I updated this type to reflect that.